pkg/ccl/crosscluster/logical/purgatory.go

```diff
@@ -152,7 +156,7 @@ func (p *purgatory) Drain(ctx context.Context) error {
 	return nil
 }
 
-func (p purgatory) Empty() bool {
+func (p *purgatory) Empty() bool {
```

Comment:
eh, why?

Reply:
hrm, the compiler was complaining at some point about the mixed receiver methods, but no longer. Fixed.